framework/auth/core.py

```diff
@@ -350,6 +366,15 @@ class User(GuidStoredObject, AddonModelMixin):
     family_name = fields.StringField()
     suffix = fields.StringField()
 
+    # OAuth2 Authorization and Authentication
+    # TODO: what are proper parameters for `DictionaryField()`
+    # TODO: what else can we store for each oauth provider?
```

Comment:
These `TODO`s can probably be removed. This may eventually be replaced with an `AbstractForeignField` pointing to some sort of OAuth provider model, but this should be able to be migrated if and when that happens